file-service-persistence/src/main/java/uk/gov/justice/services/fileservice/repository/ContentJdbcRepository.java

```diff
@@ -15,42 +18,48 @@
 /**
  * Class for handling inserts/updates/selects on the 'content' database table. This class is not
  * transactional. Each method takes a valid database connection and it is assumed that the transaction
- * would have already been started on that connection. Any failures during insert/update will throw
- * a {@link TransactionFailedException}. In which case the current transaction should be rolled
- * back
+ * would have already been started on that connection.
+ *
+ * There no update method on this class. This is because we should never update content. Delete
+ * and recreate if you need to change the file content.
+ *
+ * NB. This class does not have a unit test, but is tested instead by FilePersistenceIntegrationTest
  */
 public class ContentJdbcRepository {
 
-    private static final String SQL_FIND_BY_FILE_ID = "SELECT content FROM content WHERE file_id=? ";
-    private static final String SQL_INSERT_METADATA = "INSERT INTO content(content, file_id) VALUES(?, ?)";
-    private static final String SQL_UPDATE_METADATA = "UPDATE content set content = ? WHERE file_id = ?";
+    private static final String SQL_FIND_BY_FILE_ID = "SELECT content FROM content WHERE file_id = ? ";
+    private static final String SQL_INSERT_CONTENT = "INSERT INTO content(content, file_id) VALUES(?, ?)";
+    private static final String SQL_DELETE_CONTENT = "DELETE FROM content WHERE file_id = ? ";
 
     private final Closer closer = new Closer();
```

Comment:
What is the Closer for? Just for testing? It would be clear if we explicitly closed things, or auto-closed them as try-with-resources variables.